README.md

```diff
@@ -68,6 +66,33 @@ var DropzoneDemo = React.createClass({
 React.render(<DropzoneDemo />, document.body);
 ```
 
+Style
+=====
+
+By default, the component picks up some default styling to get you started. You can customize `<Dropzone>` by specifying a `style` and `activeStyle` which is applied when a file is dragged over the zone. You can also specify `className` and `activeClassName` if you would rather style using CSS.
+
+You have a third option : providing a function that returns the component's children.
+
+```
+<Dropzone>
+  {({ isDragActive, isDragReject }) => {
+    let color = "black", text = "Try dropping some files";
```

Comment:
Let's simplify the example. Please remove `color` from it and let's `return` some text in each of the cases just to illustrate it.